cmake/CMakeLists.txt

```diff
@@ -1166,14 +1166,8 @@ endif()
 # built; hence the ordering
 if (onnxruntime_BUILD_UNIT_TESTS)
   if (onnxruntime_ENABLE_PYTHON)
-    if(UNIX)
-      set(CMAKE_SKIP_BUILD_RPATH ON)
```

Comment:
is it safe to remove this? did you test python package?

Reply:
Yes, I tested.
If we remove this, then:
1. At build time, all the shared libraries will have RPATH embedded in.  so onnxruntime.so can find the its dependencies through absolute path.
2. When we generate the wheels, manylinux's auditwheel will read the rpath information, erase it and pack the necessary  dependency binaries in.

So, it works if we don't do anything special. 

If we keep it, we need to modify setup.py to manually pack the dependencies in, currently it doesn't work when MKML is enabled. I guess it is also true for some EPs.